README.md

```diff
@@ -23,6 +23,12 @@ contains binaries that can be run on your computer.
 
 To see the list of possible commands and arguments, run `function-runner --help`.
 
+When running this on x86 architectures (not Apple Silicon) and if you pass a Wasm file that includes debug symbols (that is, DWARF symbols), you can step debug your function with lldb by installing lldb and running:
```

Comment:
Nit: It might be clearer to add a debugging subheading. Also I think we can simplify the sentence a bit.

```suggestion
### Debugging

Wasm modules with DWARF symbols can be debugged using `lldb` on x86 architectures (not Apple Silicon). To do this, install `lldb` and run:
```